index.html

```diff
@@ -108,5 +108,83 @@ <h4>Skills</h4>
         </div>
       </div>
     </section>
+    <!----      Work section                    ---- -->
+      <section class="work-container">
+        <div class="work-tittle">
+          <h1>MY RECENT <br><br> WORK</h1>
```

Comment:
- [ ] You shouldn't use more than one `<h1>` tag in an HTML document for semantic reasons. And you have already used on in the headline section.